src/bvals/cc/flux_correction_cc.cpp

```diff
@@ -65,11 +61,11 @@ void CellCenteredBoundaryVariable::SendFluxCorrection() {
           for (int nn = nl_; nn <= nu_; nn++) {
             for (int k = pmb->ks; k <= pmb->ke; k += 2) {
               for (int j = pmb->js; j <= pmb->je; j += 2) {
-                Real amm = pco->GetFace1Area(k, j, i);
-                Real amp = pco->GetFace1Area(k, j + 1, i);
-                Real apm = pco->GetFace1Area(k + 1, j, i);
-                Real app = pco->GetFace1Area(k + 1, j + 1, i);
-                Real tarea = amm + amp + apm + app;
+                const Real amm = coords.Area(1, k, j, i);
+                const Real amp = coords.Area(1, k, j + 1, i);
+                const Real apm = coords.Area(1, k + 1, j, i);
+                const Real app = coords.Area(1, k + 1, j + 1, i);
+                const Real tarea = amm + amp + apm + app;
```

Comment:
Please use the new enums X1DIR, X2DIR, and X3DIR here and further below.

Reply:
thanks! done